docs/syntax/images.md

```diff
@@ -1,29 +1,34 @@
 # Images
 
-Images include screenshots, inline images, icons, and more. Syntax for images is like the syntax for links, with two differences:
+Images include screenshots, inline images, icons, and more. Syntax for images is like the syntax for links, with the following differences:
+
 1. instead of link text, you provide an image description
 2. an image description starts with `![` not just `[`
+3. there are restrictions on the scope of image paths
+
+:::{note}
 
-Images can be referenced from the top-level `_static` dir or a local image dir.
+Irrespective of whether you use relative or absolute paths, if you reference an image that exists in a folder that is higher in the tree than the `toc.yml` file, it is at risk of being broken if the assembler rehomes this subtree and causes a warning in the build.
```

Comment:
```suggestion
When a page is referenced in a `toc.yml` file or a `docset.yml` file and you reference an image that exists outside the tree represented in that `.yml` file, it will fail to render and will generate warnings.
```